shell/platform/windows/window.h

```diff
@@ -249,6 +253,9 @@ class Window : public KeyboardManager::WindowDelegate {
   // Accessibility node that represents an alert.
   std::unique_ptr<ui::AXPlatformNodeWin> alert_node_;
 
+  // Guard against posthumous vtable access;
+  bool vtable_is_alive = true;
```

Comment:
On a separate note, I'm surprised the linter didn't catch the missing underscore suffix on this field. Probably irrelevant given the change in approach proposed for this patch, but I'll file a bug to see why the linters aren't running on windows.